File: app/src/main/java/com/dgc/androidframe/utils/MyApplication.java
package com.dgc.androidframe.utils;

import android.app.Application;

/**
 * Created by Deng on 2018/7/26.
 */
public class MyApplication extends Application {

    public static Application app;

    @Override
    public void onCreate() {
        super.onCreate();

        app = this;
    }
}

File: app/src/main/java/com/dgc/androidframe/MyApplication.kt
package com.dgc.androidframe

import android.app.Application
import com.dgc.androidframe.utils.RuntimeContext

/**
 * Created by Deng on 2018/7/26.
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        RuntimeContext.sApplicationContext = this
    }
}
